src/main/java/com/mindera/school/spaceshiprent/emailing/EmailConsumer.java

```diff
@@ -1,4 +1,4 @@
-package com.mindera.school.spaceshiprent.components;
+package com.mindera.school.spaceshiprent.emailing;
```

Comment:
I really don't like the package name. I'd rather prefer the old one, at least as the main package for components.